app/controllers/glossaries_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# controller for glossary and entries
```

Comment:
In cases like this, where you are struggling for meaningful documentation it's best to use `#:nodoc:`, which is the appropriate way of saying you have nothing to say :)